.gitlab/notify.yml

```diff
@@ -54,10 +31,31 @@ notify-on-failure:
       # The triggering repo should already have its own notification system
       if [ "$CI_PIPELINE_SOURCE" != "pipeline" ]; then
         if [ "$DEPLOY_AGENT" = "true" ]; then
-          invoke -e pipeline.notify-failure --notification-type "deploy"
+          invoke -e pipeline.notify --notification-type "deploy"
         else
-          invoke -e pipeline.notify-failure --notification-type "merge"
+          invoke -e pipeline.notify --notification-type "merge"
         fi
       else
         echo "This pipeline was triggered by another repository, skipping notification."
       fi
+
+send_pipeline_stats:
+  stage: notify
+  # Using the buildimages image and not slack-notifier because we need Python 3.7+
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["arch:amd64"]
+  when: always
+  dependencies: []
+  script:
+    - source /root/.bashrc && conda activate ddpy3
+    - set +x
+    - export GITLAB_TOKEN=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.gitlab_read_api_token --with-decryption --query "Parameter.Value" --out text)
+    - export DD_API_KEY=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.datadog_api_key_org2 --with-decryption --query "Parameter.Value" --out text)
+    - python3 -m pip install -r tasks/libs/requirements-stats.txt
+    - |
+      # Do not send statistics if this is a child pipeline of another repo
```

Comment:
If the aim is to get statistics on failures, why does it matter if this was triggered as a child pipeline of another repo?

Reply:
That was a copy-paste from the notification jobs, I'll remove that.